packages/graphback-db-migrate/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@graphback/db-migrate",
```

Comment:
Like migrate but WDYT about `db-manage` as less specific.

Reply:
I'm on board with that :D